dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/CosmosSystemTextJSonSerializer.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// Taken from https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4332
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Cosmos;
+
+/// <summary>
+/// This class provides a default implementation of System.Text.Json Cosmos Linq Serializer.
+/// </summary>
+public class CosmosSystemTextJsonSerializer : CosmosLinqSerializer
```

Comment:
This will be added to the Azure Cosmos DB SDK soon, and we can remove this when it is.

Reply:
Can it be internal until then?

Reply:
Done.